apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/entity/pagerule/WaRuleMetadata.java

```diff
@@ -75,4 +78,27 @@ public class WaRuleMetadata {
     @Column(name = "target_type", length = 50)
     private String targetType;
 
+
+    public WaRuleMetadata(PageRuleCommand.AddPageRule command) {
+        this.ruleCd = command.ruleRequest().ruleFunction();
+        this.ruleDescText = command.ruleRequest().ruleDescription();
+        this.sourceValues = command.ruleData().sourceValues();
+        this.logic = command.ruleRequest().comparator();
+        this.sourceQuestionIdentifier = command.ruleData().sourceIdentifier();
+        this.targetQuestionIdentifier = command.ruleData().targetIdentifiers();
+        this.targetType = command.ruleRequest().targetType();
+        this.addTime = command.addTime();
+        this.addUserId = command.userId();
+        this.lastChgTime = command.lastChangeTime();
+        this.recordStatusCd = ACTIVE;
+        this.lastChgUserId = command.userId();
+        this.recordStatusTime = command.recordStatusTime();
```

Comment:
Suggestion: most of the other commands have a `requestedOn` field can be used for both the `addTime` and `recordStatusTime`